text/src/autogluon/text/automm/constants.py

```diff
@@ -75,3 +75,15 @@
 UNION_SOUP = 'union_soup'
 GREEDY_SOUP = 'greedy_soup'
 BEST_SOUP = 'best_soup'
+
+# registered model keys
+CLIP = "clip"
+TIMM_IMAGE = "timm_image"
+HF_TEXT = "hf_text"
+NUMERICAL_MLP = "numerical_mlp"
+CATEGORICAL_MLP = "categorical_mlp"
+FUSION_MLP = "fusion_mlp"
+AVAILABLE_MODELS = [
+    CLIP, TIMM_IMAGE, HF_TEXT, NUMERICAL_MLP,
+    CATEGORICAL_MLP, FUSION_MLP,
+]
```

Comment:
Now adding a new model will require revising several files. We may later need a document about how to add a new models. Potentially marked as a TODO item.

Reply:
Marked.